data/campaigns/Northern_Rebirth/scenarios/11a_The_Eastern_Flank.cfg

```diff
@@ -343,17 +343,17 @@
 
         [message]
             speaker=Rakshas
-            message= _ "What?! YOU! YOU ANNOYING, DISGUSTING LITTLE VERMIN! DON’T YOU KNOW HOW TO DIE?!"
+            message= _ "What?! YOU! YOU ANNOYING, DISGUSTING LITTLE PEST! DON’T YOU KNOW HOW TO DIE?!"
         [/message]
 
         [message]
             speaker=Tallin
-            message= _ "Foul one, the only one who will be dying here is you and anyone who comes between us. Stand and fight!"
+            message= _ "Foul one, the only one who will be dying here is you. Stand and fight!"
         [/message]
 
         [message]
             speaker=Rakshas
-            message= _ "Bah! I have better things to do than stamp out your insignificant life. Generals, kill him and bring me his head!"
+            message= _ "Bah! I have better things to do than stamp out your insignificant life. Generals, bring me his head!"
```

Comment:
While 'head' implies not attached to the body - and so killing is a prerequisite - I think I prefer the aggressive tone of the original 'kill' order, even if it's redundant.

Reply:
That might be true in other cases, but given Rakshas' current mood, it would be improper to waste time on redundant words.

Reply:
I'd think given his mood he'd be all the more inclined to speak improperly, but whatever.